src/App.tsx

```diff
@@ -125,6 +126,11 @@ function App() {
 
   const { grants } = grantData;
   const { projects } = projectData;
+  const [giveAssetType, setGiveAssetType] = useState<"sOHM" | "gOHM">("sOHM");
+
+  const changeGiveAssetType: ChangeAssetType = (checked: boolean) => {
+    setGiveAssetType(checked ? "gOHM" : "sOHM");
+  };
```

Comment:
I'm wondering if this should move outside of `app.tsx`. small nit, but would be nice to isolate this inside of the give files. You could put it inside a hook that gets called in the give views... but... that may be messy. lmk what you think.

Reply:
I agree that it's best for it to not be in `app.tsx`. let me try something to keep it just in the give files and I'll let you know if it's too messy

Reply:
first thing I tried didn't work smoothly, have one more implementation I can think of that I'll try to do tomorrow

Reply:
I agree this should be refactored. In my brief review it looks like the challenge is that this setting needs to be shared between multiple give routes.

Instead of treating App as the catchall for sharing state between child components, there should be a parent level component for Give that handles this for you. 
We handled a similar case with Wallet by using switch statements to route to the appropriate sub component. We still have routes setup in app.tsx, but we're only pointing to a component.
Wallet index.tsx handles any shared state/prop passing to avoid unnecessary clutter in app.tsx and keep things isolated.

It's additional work, but it'll keep things cleaner in the longrun.

Reply:
ok, as discussed elsewhere. We'll push this refactor into a follow-up next week. @0xLienid